app/views/api_keys/index.html.erb

```diff
@@ -0,0 +1,59 @@
+<% @title = t(".api_keys") %>
+
+<div class="t-body">
+  <div class="table">
+    <div class="row row__header">
+      <div class="cell">
+        <%= t(".name") %>
+      </div>
+      <div class="cell">
+        <%= t(".scope") %>
+      </div>
+      <div class="cell">
+        <%= t(".created_at") %>
+      </div>
+      <div class="cell">
+      </div>
+    </div>
+
+    <% for api_key in @api_keys do %>
+      <div class="row">
+        <div class="cell">
+          <%= api_key.name %>
+        </div>
+        <div class="cell">
+          <%= i18n_api_scopes api_key %>
+        </div>
+        <div class="cell">
+          <%= t(".time_ago", time: time_ago_in_words(api_key.created_at)) %>
```

Comment:
Can we provide the exact date/time in the alt field? I find that sometimes having human friendly time ago is not useful